priv/templates/embedded/bundle-install.greenbar

```diff
@@ -0,0 +1 @@
+Bundle "~$results[0].name~" version "~$results[0].version~" installed.
```

Comment:
We should wrap this in an `~each var=$results~` tag to handle cases where people install a bunch of bundles at once:
```
for bundle in heroku pingdom twitter | bundle install $bundle
```
Otherwise, we'll just show the installation of the first bundle.

(Alternatively, you could do a table for the more-than-one bundle scenario; that might be overkill in this case, though.)